taqueria-vscode-extension/src/lib/pure.ts

```diff
@@ -155,7 +71,7 @@ export class TaqifiedDir {
 				Promise.reject({
 					code: 'E_NOT_TAQIFIED',
 					pathProvided: dir,
-					msg: 'The given directory is not taqified as its missing a .taq directory.',
+					msg: "The given directory is not taqified as it's missing a .taq directory.",
```

Comment:
`The current directory is not a Taqueria project. Run `taq init` to Taq'ify it`

Reply:
This is part of the formatting. May be out of scope for this PR?

Reply:
Possibly formatting. I'd prefer to not touch this as it may complicate merging.

Reply:
The change is from another PR. Should be fixed there or in a new PR.